src/decode_gps_l1.c

```diff
@@ -112,10 +113,11 @@ static void decoder_gps_l1_process(const decoder_channel_info_t *channel_info,
   /* Decoded a new ephemeris. */
   ephemeris_new(&e);
 
-  ephemeris_t *eph = ephemeris_get(channel_info->sid);
-  if (!eph->valid) {
+  u8 v, h;
+  ndb_ephemeris_info(channel_info->sid, &v, &h, NULL, NULL);
+  if (!v) {
     char buf[SID_STR_LEN_MAX];
     sid_to_string(buf, sizeof(buf), channel_info->sid);
-    log_info("%s ephemeris is invalid", buf);
```

Comment:
Actually I don't even think we should be checking this here. Maybe just call `ephemeris_new()` and delete the stuff after.